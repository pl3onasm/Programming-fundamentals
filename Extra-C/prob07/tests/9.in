20
?0???11???01?10?0?10
