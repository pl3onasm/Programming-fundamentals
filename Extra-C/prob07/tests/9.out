00100110010101010010
00100110010101010110
00100110100101010010
00100110100101010110
00100110110101010010
00100110110101010110
00110110010101010010
00110110010101010110
00110110100101010010
00110110100101010110
00110110110101010010
00110110110101010110
10010110010101010010
10010110010101010110
10010110100101010010
10010110100101010110
10010110110101010010
10010110110101010110
10100110010101010010
10100110010101010110
10100110100101010010
10100110100101010110
10100110110101010010
10100110110101010110
10110110010101010010
10110110010101010110
10110110100101010010
10110110100101010110
10110110110101010010
10110110110101010110
